tests/Test/IntegrationCase.php

```diff
@@ -120,13 +120,6 @@ public function getInputCode()
      */
     public function getRequirement($name)
     {
-        if (!\is_string($name)) {
```

Comment:
:-1:

Reply:
Same reasoning: it's PHPStan's responsibility now.